android/app/src/main/java/com/example/gutenbergkit/MainActivity.kt

```diff
@@ -41,6 +78,88 @@ class MainActivity : AppCompatActivity() {
             .setCookies(emptyMap())
             .build()
 
-        gbView.start(config)
+    private fun createRemoteConfiguration(config: ConfigurationItem.RemoteEditor): EditorConfiguration =
+        createCommonConfigurationBuilder()
+            .setPlugins(true) // Enable plugins for remote editor
+            .setSiteURL(config.siteUrl)
+            .setSiteApiRoot(config.siteApiRoot)
+            .setSiteApiNamespace(arrayOf("wp/v2"))
```

Comment:
Setting a namespace is applicable for WPCOM sites communicating through the WPCOM API. We do not need to set a namespace for the default `wp/v2`. 
```suggestion
            .setSiteApiNamespace(arrayOf())
```